docs/BarChart/README.md

```diff
@@ -0,0 +1,79 @@
+# Bar Chart
+
+## Installation
+
+Backpack Android is available through [Jitpack](https://jitpack.io/#Skyscanner/backpack-android). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
+
+The Bar Chart component can be used in both XML and Kotlin/Java
```

Comment:
Since it can't really be used only in XML I think we should change this to clarify it a bit.

E.g. The Bar Chart component can be used in both XML and Kotlin/Java, however, feeding the component with data can only be done programatically.

Reply:
Done!